instrumentation/kafka-clients/src/test/java/brave/kafka/clients/ITKafkaTracing.java

```diff
@@ -331,19 +301,19 @@ void checkB3Unsampled(ConsumerRecords<String, String> records) {
       });
   }
 
-  @Test public void customSampler_consumer() throws Exception {
+  @Test public void customSampler_consumer() {
     String topic = testName.getMethodName();
 
-    consumerTracing = KafkaTracing.create(MessagingTracing.newBuilder(
-      Tracing.newBuilder().spanReporter(consumerSpans::add).build()
-    ).consumerSampler(MessagingRuleSampler.newBuilder()
-      .putRule(operationEquals("receive"), Sampler.NEVER_SAMPLE)
-      .build()).build());
+    consumerTracing = KafkaTracing.create(
+      MessagingTracing.newBuilder(consumerTracing.messagingTracing.tracing())
```

Comment:
this sort of stuff looks gross now, but when we complete the messaging abstraction, should end up contained.